kornia/models/detector/rtdetr.py

```diff
@@ -0,0 +1,140 @@
+import warnings
+from typing import Optional
+
+import torch
+from torch import nn
+
+from kornia.contrib.models.rt_detr import DETRPostProcessor
```

Comment:
Why don’t we move it in this new module ?

Reply:
Lots of things will break, some testing or maybe documentation. I do not want to touch it for now.